### DISCUSSION TITLE
Score bigger then 1

### DISCUSSION THREAD

**Discussion by bleekk** [krisk/Fuse#626] [Q&A] [answered]
The documentation says that the score is between 0 and 1. 
0 should be the perfect match.
But when I run it, my perfect match has a score of 9.287......

Is this the right behavior?

**Comment by bleekk** [comment]
Even in the demo the score for exact matching words is 9.287......

**Comment by krisk** [comment, marked as answer]
The score is `9.287439764962262e-10`, which is `0.0000000009287439764962262` 😄
